IBPSA/Fluid/Movers/PrescribedFlow.mo

```diff
@@ -0,0 +1,78 @@
+within IBPSA.Fluid.Movers;
```

Comment:
I think this belongs in the Airflow package since it's only used there?

Reply:
I wasn't sure yet because looking at the component  itself it is not actually restricted to airflow only.